webviz_config/_oauth2.py

```diff
@@ -4,6 +4,7 @@
 
 import msal
 import flask
+from werkzeug.middleware.proxy_fix import ProxyFix
```

Comment:
Maybe we technically should add `werkzeug` as a dependency now? (Even though I guess `flask` does not have any immediate plans of removing the `werkzeug` dependency :laughing: )

Reply:
Explicit is always a good idea. Of course, if flask drops using werkzeug, this workaround will also fail immediately.  Side note: Flask is also not specified as an explicit dependency